core/pom.xml

```diff
@@ -602,6 +608,12 @@ THE SOFTWARE.
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
+      <exclusions>
+        <exclusion> <!-- pick up from Stapler -->
+          <groupId>com.google.code.findbugs</groupId>
+          <artifactId>jsr305</artifactId>
```

Comment:
Maybe it's safer to just define the newest version in this POM instead of adding exclusions